### DISCUSSION TITLE
Change order of filters in object browser

### DISCUSSION THREAD

**Discussion by cunninghammike1125** [codefori/vscode-ibmi#848] [General]
How do I change the order of filters in the object browser.  I'd like to group our development, UAT and production filters together.

**Comment by worksofliam** [comment]
Order cannot be changed through the UI, but you can manually edit the JSON settings to re-order.

I would recommend you look into using profiles instead. You can create a profile for each environment. Each profile has its own filters, library list, and others.

https://halcyon-tech.github.io/vscode-ibmi/#/pages/settings/profiles

**Comment by cunninghammike1125** [comment]
Ok.  Thank you for the tip.  Can I change the order of the profiles?

On Wed, Aug 31, 2022 at 2:53 PM barry ***@***.***> wrote:

> Order cannot be changed through the UI, but you can manually edit the JSON
> settings to re-order.
>
> I would recommend you look into using profiles instead. You can create a
> profile for each environment. Each profile has its own filters, library
> list, and others.
>
> https://halcyon-tech.github.io/vscode-ibmi/#/pages/settings/profiles
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/halcyon-tech/vscode-ibmi/discussions/848#discussioncomment-3521168>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQBLAEZHW4QUNA364JECYTV36Z5LANCNFSM6AAAAAAQBVSAWY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

Mike Cunningham
Senior Programmer / Analyst
(312) 498-6291

**Reply by worksofliam** [reply]
Not right now, but that might be something we can add easily.

**Comment by cunninghammike1125** [comment]
Ok, thanks.

On Thu, Sep 1, 2022 at 11:30 AM barry ***@***.***> wrote:

> Not right now, but that might be something we can add easily.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/halcyon-tech/vscode-ibmi/discussions/848#discussioncomment-3528727>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQBLAEGSVWJ3WGZ44SUM53V4DKYZANCNFSM6AAAAAAQBVSAWY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

Mike Cunningham
Senior Programmer / Analyst
(312) 498-6291
